lib/routes/nbt/sjxy/index.ts

```diff
@@ -0,0 +1,70 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import got from '@/utils/got';
+
+const host = 'https://sjxy.nbt.edu.cn';
+const typeMap = {
+    notice: '/index/tzgg.htm',
+    news: '/index/xwdt.htm',
+};
+
+const titleMap = {
+    notice: '通知公告',
+    news: '新闻动态',
+};
+
+export const route: Route = {
+    path: '/sjxy/:type/',
+    categories: ['university'],
+    example: '/nbt/sjxy/notice',
+    parameters: { type: '通知类型，默认为通知公告' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '通知公告',
+    maintainers: ['A-normal'],
+    handler,
+    description: `| 通知公告 | 新闻动态 |
+    | ---------- | ----- |
+    | notice     | news  |`,
```

Comment:
```suggestion
  | ---------- | ----- |
  | notice     | news  |`,
```
Indent with 2 or 0 spaces.